compat/src/index.js

```diff
@@ -66,7 +66,10 @@ function cloneElement(element) {
  */
 function unmountComponentAtNode(container) {
 	if (container._children) {
-		preactRender(null, container);
+		createRoot(container).render(null);
+		return true;
+	} else if (container._root) {
+		container._root.render(null);
 		return true;
 	}
 	return false;
```

Comment:
I think this may compile to smaller output:

```js
function unmountComponentAtNode(container) {
	let root = container._root || (container._children && createRoot(container));
	return root && (root.render(null), true);
}
```